### PR TITLE
Change keyboard behavior on Discover Search

### PR DESCRIPTION
**Pull request by osdnk**
Make it not restoring on scroll to top as discussed with @christianbaroni.

### CODE REVIEW

**Review by skylarbarrera — APPROVED** [review]
This looks like it works as expected, may want @christianbaroni to take a look


Rerunning CI

**Comment by osdnk** [comment]
oops, let's see now

File: src/components/discover-sheet/DiscoverSearchContainer.js [hunk 1]
```diff
@@ -63,7 +64,22 @@ export default forwardRef(function DiscoverSearchContainer(
     [setIsSearchModeEnabled, setShowSearch, setViewPagerSwipeEnabled]
   );
 
-  upperContext.onFabSearch.current = setIsInputFocused;
+  const onTapSearch = useCallback(() => {
+    if (isSearchModeEnabled) {
+      sectionListRef.current.scrollToLocation({
```
Comment (brunobar79) [inline src/components/discover-sheet/DiscoverSearchContainer.js:69]: .current?

File: src/components/discover-sheet/DiscoverSearchContainer.js [hunk 2]
```diff
@@ -63,7 +64,22 @@ export default forwardRef(function DiscoverSearchContainer(
...
+        animated: true,
+        itemIndex: 0,
+        sectionIndex: 0,
+        viewOffset: 0,
+        viewPosition: 0,
+      });
+      searchInputRef.current.focus();
```
Comment (brunobar79) [inline src/components/discover-sheet/DiscoverSearchContainer.js:76]: Maybe here too

**Review by brunobar79 — APPROVED** [review]
💯

File: src/components/discover-sheet/DiscoverSearchContainer.js (hunk 1 above)
Comment (osdnk) [reply src/components/discover-sheet/DiscoverSearchContainer.js:69]: There's no need for that! This will be always present

**Pull request merged**